README.md

```diff
@@ -273,8 +317,12 @@ Apache 2.0 - See [COPYING](COPYING) for more information.
 [cloud-pubsub]: https://cloud.google.com/pubsub/
 [cloud-pubsub-docs]: https://cloud.google.com/pubsub/docs
 
+[cloud-search]: https://cloud.google.com/search/
+[cloud-search-docs]: https://cloud.google.com/search/
```

Comment:
Yeah :) None of the docs are fully public yet, and there isn't yet a landing page like the others.